docs/run/index.md

```diff
@@ -8,7 +8,19 @@ parent:
 
 # Start a Blockchain
 
-One of your first actions is to start your blockchain. To start your blockchain, use the `starport serve` command.
+Blockchains are decentralized applications that in production run (often) the same software on many validator nodes by different people/entities. To launch a blockchain in production entities coordinate on the launch process together and start their nodes.
+
+During development, however, a blockchain can be started locally on a single validator node. This is very convenient, because you can quickly restart a chain and iterate faster. Starting a chain on single node in development is similar to starting a traditional web application on a local server.
```

Comment:
```suggestion
Blockchains are decentralized applications (dApps).

- In production, dApps often run the same software on many validator nodes that are run by different people and entities. To launch a blockchain in production, the validator entities coordinate the launch process to start their nodes simultaneously.
- During development, a blockchain can be started locally on a single validator node. This convenient process lets you can restart a chain quickly and iterate faster. Starting a chain on a single node in development is similar to starting a traditional web application on a local server. 
```

Reply:
hmmm what do we mean here?

> coordinate on the launch process together and start their nodes

I don't know if we really mean simultaneously? (the same time?) 

> the validator entities coordinate the launch process to start their nodes simultaneously.

My main contribution here is to provide parallel sentence structure for the comparison of the production vs development. let's talk it through if needed, we can make this crystal clear.

Reply:
Removed "dApps", because even though it's short for "decentralized applications", the meaning for me is different. Blockchain is a decentralized application, but dApp is a user-facing app that is powered by one or more chains or contracts. It's confusing, I know.

Reply:
> I don't know if we really mean simultaneously? (the same time?)

Pretty much at the same time.